rust/opendal/opendal.go

```diff
@@ -0,0 +1,63 @@
+package opendal
+
+import (
+	"github.com/goplus/llgo/c"
+	_ "unsafe"
+)
+
+// Write the .pc file for the dylib generated by the Rust library and copy it to pkg-config for proper location.
+const (
+	LLGoPackage = "link: $(pkg-config --libs opendal_c); -lopendal_c"
+)
+
+type Opendal_result_operator_new struct {
```

Comment:
Rename `Opendal_result_operator_new` to `ResultOperatorNew` and remove the `Opendal_` prefix from both function and structure names."